### PR TITLE
Add Oskari style documentation

### PR DESCRIPTION
**Pull request by atlain**
Definition/example document for Oskari style

### CODE REVIEW

File: md/documentation/examples/oskari-style.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    },
+    "image": { // image style
+        "shape": "marker.png", // external icon
+        "size": 3, // Oskari icon size.
+        "sizePx": 20, // Exact icon px size. Used if 'size' not defined.
+        "offsetX": 0, // image offset x
+        "offsetY": 0, // image offset y
+        "opacity": 0.7, // image opacity
+        "radius"
+        2, // image radius
```
Comment (ZakarFin) [inline md/documentation/examples/oskari-style.md]: Missing : between radius and 2?

File: md/documentation/examples/oskari-style.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
+# Oskari style
+
+JSON style object for styling layers. See example usages in AddFeaturesToMapRequest or Vector Tile Layer Plugin. VisualizationForm component also supports getting and setting style values in Oskari style format.
+
```
Comment (ZakarFin) [inline md/documentation/examples/oskari-style.md]: I think all of these are optional right? We should mention it here.

File: md/documentation/examples/oskari-style.md (hunk 1 above)
Comment (atlain) [reply md/documentation/examples/oskari-style.md]: Yep, I'll fix it

File: md/documentation/examples/oskari-style.md (hunk 2 above)
Comment (atlain) [reply md/documentation/examples/oskari-style.md]: That text was supposed to be just some sort of descriptive lead and examples. What do you suggest?

Comment (ZakarFin) [reply md/documentation/examples/oskari-style.md]: I was just thinking of a short note that every field in the json is optional and can be omitted when defining a style.

Comment (atlain) [reply md/documentation/examples/oskari-style.md]: Oh yes, I was just confused by what you meant by "all of these". I'll add a short note.

**Pull request merged**